README.md

```diff
@@ -13,35 +13,47 @@ Keep calm and add Geocoder to your `mix.exs` dependencies:
 
 ```elixir
 def deps do
-  [{:geocoder, "~> 0.7"}]
+  [{:geocoder, "~> 1.0"}]
```

Comment:
Hm. Unfortunately we don't have the most up to date version on hex.pm, yet. @knrz any way you could grant me access to the hex.pm package so that I can craft releases?